src/main.rs

```diff
@@ -303,6 +325,26 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 Err(_) => "".to_owned(),
             }
         }
+        Command::PostComment { message, secret } => {
+            let client = reqwest::blocking::Client::new();
+            let github_event_path = env::var("GITHUB_EVENT_PATH")?;
+            let v: Value = serde_json::from_str(&github_event_path)?;
+            let uri = v["pull_request"]["comments_url"].as_str();
+
+            fn construct_headers() -> HeaderMap {
+                let mut headers = HeaderMap::new();
+                headers.insert(USER_AGENT, HeaderValue::from_static("action-cli"));
```

Comment:
Good idea. It would also be nice if it included the client version, so `action-cli/<version>`